.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "typescript.preferences.importModuleSpecifier": "relative"
+  "typescript.preferences.importModuleSpecifier": "relative",
+  "[typescript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
+  }
```

Comment:
I would keep this in a separate PR just for keeping this one with the unit test.

What does this change do exactly?

Reply:
I'm afraid it was vscode, I didn't realise

Reply:
I've just undo this change